components/build/jvm-build-service/system-config.yaml

```diff
@@ -1,26 +1,5 @@
 ---
-apiVersion: v1
-kind: ConfigMap
+apiVersion: jvmbuildservice.io/v1alpha1
+kind: SystemConfig
 metadata:
-  name: jvm-build-system-config
-  namespace: jvm-build-service
-data:
-  image.cache: quay.io/redhat-appstudio/hacbs-jvm-cache:cd568e1d98ac04ac1f4f982c702732df7951b920
-  builder-image.names: jdk11,jdk8,jdk17
-  builder-image.jdk8.image: quay.io/redhat-appstudio/hacbs-jdk8-builder:a4787c4c72c7f606515e19dc3057e79779b4f5a1
-  builder-image.jdk11.image: quay.io/redhat-appstudio/hacbs-jdk11-builder:a4787c4c72c7f606515e19dc3057e79779b4f5a1
-  builder-image.jdk17.image: quay.io/redhat-appstudio/hacbs-jdk17-builder:a4787c4c72c7f606515e19dc3057e79779b4f5a1
-  builder-image.jdk8.tags: jdk:8,maven:3.8,gradle:7.4.2;6.9.2;5.6.4;4.10.3
-  builder-image.jdk11.tags: jdk:11,maven:3.8,gradle:7.4.2;6.9.2;5.6.4;4.10.3
-  builder-image.jdk17.tags: jdk:17,maven:3.8,gradle:7.4.2;6.9.2
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: jvm-build-config
-  namespace: jvm-build-service
-data:
-  enable-rebuilds: "false"
-  maven-repository-300-jboss: "https://repository.jboss.org/nexus/content/groups/public/"
-  maven-repository-301-gradle: "https://repo.gradle.org/artifactory/libs-releases"
-  maven-repository-302-gradleplugins: "https://plugins.gradle.org/m2"
+  name: cluster
```

Comment:
introduced this empty systemconfig to allow for overrides, but still manage the defaults out of the jvm-build-service repo, which was @stuartwdouglas 's preference as I understood it from our last conversations on this